app/src/main/java/fr/free/nrw/commons/utils/ImageUtils.java

```diff
@@ -43,7 +43,11 @@
     public static final int IMAGE_WAIT = -2;
     public static final int EMPTY_TITLE = -3;
     public static final int FILE_NAME_EXISTS = -4;
+    public static final int FILE_FBMD = -5;
     static final int NO_CATEGORY_SELECTED = -5;
+    /*
+    * The parameter FILE_FBMD is returned from the class ReadFBMD if the uploaded image contains FBMD data else returns IMAGE_OK
+    * */
```

Comment:
Please move this comment to immediately before the variable declaration, and make it a real Javadoc with `**`. That way, it will appear in IDE tooltips etc.